server/controllers/calls.js

```diff
@@ -107,6 +108,33 @@ function afterPutCallAttempt(res, outcome, contact_id, attempt_num, campaign_id)
   }
   return res.status(200).json({ message: 'call log successfully updated' });
 }
+
+function handleHangUpFlow(res, user_id, call_id, newStatus) {
```

Comment:
Why do we need to accept a newStatus param when we really just always use status HUNG_UP as the name of the method suggests? I think it'll be cleaner if we remove it.

Reply:
Ture ture, I'll get on it.

Reply:
O.o *true